docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx

```diff
@@ -0,0 +1,941 @@
+
+1. Setting up an account for 100ms
+2. Stream Dashboard integration
+3. Configure dependencies
+4. Implement a ViewModel for video calls
+5. Implement video call layouts
+6. Implement custom call messages using custom attachments
+
+:::note
+This tutorial assumes you already use [Stream Chat SDK for Android](https://getstream.io/chat/sdk/android/) in your Android project. So you will not cover how to set up the Stream Chat SDK in this tutorial. For more information, check out the [Android Chat Messaging Tutorial](https://getstream.io/tutorials/android-chat/) guide.
+:::
+
+## 1. Setting Up an Account for 100ms
+
+First, let’s go over a quick introduction to [100ms](https://www.100ms.live). It is a service that allows you to do video conferencing, audio, and more. Their aim is to provide you with a wide range of extensible features, all while allowing you to get started quickly with minimum effort.
+
+To get started, you must [set up an account](https://dashboard.100ms.live/register) for the platform – click the **Try For Free** button for a trial to use for this tutorial. You can sign up with either a Google or GitHub account, or you can use any other email address. You will receive an email asking you to confirm your credentials.
+
+Next, you’ll get a quick tour of how to create your own video conference. Here is an outline of the steps you must take:
+
+1. Choose a template: Select **Video Conferencing**, hit **Next**
+2. Add a few more details: Enter everything that is valid for you
+3. Choose a subdomain: Create a subdomain that is suitable for your use case and select the closest region (e.g. in our case, “integrationguide” and “EU” make the most sense, resulting in the domain: **integrationguide.app.100ms.live**)
+4. Your app is ready: You can join the room if you want to see a sample (not necessary)
+
+From here, click the **Go to Dashboard** button at the bottom. After completing the quick introductory tour, your account and app will be ready to continue. Nice job!
+
+You will come back to the Dashboard later, but we will move on to other steps next.
+
+## 2. Stream Dashboard Integration
+
+In order for the integration of 100ms to work there needs to be a server component handling token generation and more. Normally this would require a custom server implementation but Stream offers first-class integration for 100ms relieving you of these duties.
+
+There are only a few setup steps to go through in the Stream dashboard and this guide details all of them:
+
+1. Head over to the [Dashboard](https://dashboard.getstream.io) and login
+2. Create a new app or select your app by name
+3. In the sidebar on the left, select **Ext. Video Integration**
+4. Make sure the **100ms** tab is selected
+
+This is the screen that you navigated to:
+
+![View of the Stream Dashboard when an app was selected with the external video integration.](../../../assets/hms-guide-before.png)
```

Comment:
I believe the correct path is: `../../assets/hms-guide-before.png`

Reply:
The same goes for all the images. I tested with: 
```
cd [path/to/folder/of/mdx-document]
open [path/of/link/of/image]
```

The command `open` only works when removing one `../`

Reply:
Nice catch! Thanks for check things ❤️  
Done: [f589c65](https://github.com/GetStream/stream-chat-android/pull/4181/commits/f589c65353b7c5296f4a30f331fed513d67e8866)